layouts/page/cuml-accel.html

```diff
@@ -0,0 +1,329 @@
+{{- define "main" -}}
+
+<!-- Hero Partial -->
+{{ partial "hero.html" (dict
+"text" "GPU Accelerated Machine Learning"
+"btn_text" "Get Started"
+"btn_link" "#get-started"
+) }}
+
+<!-- Intro -->
+<section class="row section mb-6">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="text-center mb-10"> OPEN BETA: ACCELERATING SCIKIT-LEARN, UMAP, <br> AND HDBSCAN WITH ZERO CODE 
+        CHANGE </h1>
+      <div class="row">
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-light fa-file-binary"></i> Zero Code Change Acceleration </h2>
+          <p>Write your code with the ease and familiarity of scikit-learn, UMAP, or HDBSCAN. Just load 
+            <strong>cuml.accel</strong> and run your code on the GPU for maximum performance.</p>
+
+          <br>
+
+          <h2 class="mt-4"><i class="fa-light fa-chart-diagram"></i> Algorithms for Every Use Case </h2>
+          <p>Accelerating the most popular estimators for classification, regression, dimensionality reduction, and 
+            clustering.</p>
+        </div>
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-light fa-gauge-max"></i> Train Models Faster </h2>
+          <p>Up to 50x faster training times for popular machine learning algorithms. Spend less time waiting and more 
+            time optimizing your model.</p>
+
+          <br>
+
+          <h2 class="mt-4"><i class="fa-light fa-laptop-binary"></i> Device Independent Code </h2>
+          <p>Develop on GPUs and deploy on CPUs. Or vice versa. All without any changes to your code.</p>
```

Comment:
Should these be swapped? I think more users would develop on CPU with this and the production deploys use GPUs.
```suggestion
          <p>Develop on CPU and deploy on GPUs. Or vice versa. All without any changes to your code.</p>
```